Docs/user_guide/coordinate_systems.md

```diff
@@ -0,0 +1,268 @@
+# Coordinate systems
+
+## Introduction
+
+One of the issues while dealing with medical images and applications are the
+differences between the coordinate systems. There are three coordinate systems
+commonly used in imaging applications: a difference can be made between the
+**world**, **anatomical** and the **image coordinate system**.
+
+The following figure illustrates the three spaces and their corresponding
+axes:
+
+![coordinate_systems](https://github.com/Slicer/Slicer/releases/download/docs-resources/coordinate_systems.png){.bg-warning w=600px align=center}
```

Comment:
```suggestion
![coordinate_systems](https://github.com/Slicer/Slicer/releases/download/docs-resources/coordinate_systems.png){w=600px align=center}
```

The attribute `.bg-warning` was an example of attribute and is not needed here.

Reply:
Sorry, thanks for the notice. Removed in [0b7a79e](https://github.com/Slicer/Slicer/commit/0b7a79ed1b791335814ae637ee267ba8edf3750b).